widgets/social-media-buttons/tpl/default.php

```diff
@@ -6,7 +6,19 @@
 		if( !empty($instance['design']['hover']) ) $classes[] = 'ow-button-hover';
 		$classes[] = "sow-social-media-button-" . sanitize_html_class( $network['name'] );
 		$classes[] = "sow-social-media-button";
-		$title = empty( $network['icon_title'] ) ? sprintf( __( '%s on %s', 'so-widgets-bundle' ), get_bloginfo( 'name' ), ucwords( str_replace( '-', ' ', $network['name'] ) ) ) : $network['icon_title'];
+
+		if( ! empty( $network['icon_title'] ) ) {
+			$title = $network['icon_title'];
+		} else {
+			if ( $network['name'] == 'email' || $network['name'] == 'phone' ) {
+				if ( $network['title'] == 'envelope' ) $network['name'] == 'email';
+
+				/* translators: Social Media Button with a verb and then site name. As an example, Call Eastlink Windows */
+				$title = sprintf( __( '%s %s', 'so-widgets-bundle' ), ucfirst( $network['name'] ), get_bloginfo( 'name' ) );
```

Comment:
I think the below format could work for email and phone as is, e.g. Eastlink Windows on Email, but regardless, perhaps we should add a field to let users specify the format they want. Then we can just have our one as a default.

Reply:
We do actually already have a way for users to specify a custom title, icon_title. The issue with that though is we can't expect users to always fill that field in just to correct the title - for example, "Envelope on Eastlink Windows" is obviously wrong.

With that said, I do like the Sitename on type name a little more than "Email Eastlink Windows", but the problem with that is that both buttons use tel and mailto. By clicking that link you're not taken to somewhere, you're doing something - be it calling or emailing.